server/protocols/webadmin-cli/src/test/java/org/apache/james/cli/DomainManageTest.java

```diff
@@ -54,20 +54,22 @@ void domainListCommandShouldWShowOnlyDefaultDomain(GuiceJamesServer server) {
 
         int exitCode = WebAdminCli.executeFluent(new PrintStream(outputStreamCaptor), new PrintStream(errorStreamCaptor),
                 "--url", "http://127.0.0.1:" + port.getValue(), "domain", "list");
+
         assertThat(exitCode).isEqualTo(0);
         assertThat(outputStreamCaptor.toString().trim().toCharArray()).containsOnly("localhost".toCharArray());
     }
 
     @Test
-    void domainCreateCommandWithValidNameShouldSuccessfully(GuiceJamesServer server) {
+    void domainCreateCommandWithValidNameShouldSucceed(GuiceJamesServer server) {
         Port port = server.getProbe(WebAdminGuiceProbe.class).getWebAdminPort();
 
         int exitCode = WebAdminCli.executeFluent(new PrintStream(outputStreamCaptor), new PrintStream(errorStreamCaptor),
             "--url", "http://127.0.0.1:" + port.getValue(), "domain", "create", "linagora.com");
-        assertThat(exitCode).isEqualTo(0);
 
         WebAdminCli.executeFluent(new PrintStream(outputStreamCaptor), new PrintStream(errorStreamCaptor),
             "--url", "http://127.0.0.1:" + port.getValue(), "domain", "list");
+
+        assertThat(exitCode).isEqualTo(WebAdminCli.CLI_FINISHED_SUCCEED);
```

Comment:
We tend to not reuse production constant in test (as modifying the constant itself will lead to an invalid result without breaking the test)